indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -308,6 +309,16 @@ public static void injectSystemProperties(Job job)
   {
     injectSystemProperties(job.getConfiguration());
   }
+  public static void injectDruidProprties(Configuration configuration) {
+    String mapJavaOpst = configuration.get(MRJobConfig.MAP_JAVA_OPTS);
+    String reduceJavaOpts = configuration.get(MRJobConfig.MAP_JAVA_OPTS);
+    for (String propName: System.getProperties().stringPropertyNames()){
+      if (propName.startsWith("druid.storage.")) {
```

Comment:
In view of #4180 and similar, what do you think of making it configurable which props get passed down to the MR JVMs? Maybe similar to `druid.indexer.runner.allowedPrefixes`, have a `druid.indexer.hadoopProperties.allowedPrefixes` or similar and have `druid.storage.` always be an allowed prefix or be the default